### PR TITLE
Add Twitter's bootstrap toolkit

### PR DESCRIPTION
**Pull request by rricard**
I just wanted support for [bootstrap](https://github.com/twitter/bootstrap/) in asset.

You can now require `bootstrap` and `bootstrap-less` (the less source version) in your project !

### CODE REVIEW

File: assets.json [hunk 1]
```diff
@@ -17,6 +17,29 @@
     ],
     "version": "1.0"
   },
+  "bootstrap": {
+    "url": "https://github.com/twitter/bootstrap/raw/master/bootstrap-1.1.1.css",
```
Comment (tj) [inline assets.json]: not sure if you want to add {version} in there

Comment (tj) [inline assets.json]: but cool :D thanks

Comment (rricard) [inline assets.json]: I've already tried this in fact :
`https://github.com/twitter/bootstrap/raw/v{version}/bootstrap-{version}.css`

But it downloads and calls the file `bootstrap-{version}.css` in my public folder ...

Maybe I can try to modify this behavior. I'll see it later

Comment (tj) [inline assets.json]: ohhhh I probably have a bug doing one replace only, I'll fix that then, thanks!

**Comment by rricard** [comment]
I've fixed the {version} bug. Now, everything seems to be ok !

**Pull request merged**